frmts/pdf/CMakeLists.txt

```diff
@@ -21,19 +21,19 @@ target_include_directories(gdal_PDF PRIVATE ${GDAL_RASTER_FORMAT_SOURCE_DIR}/vrt
                                             ${GDAL_VECTOR_FORMAT_SOURCE_DIR}/mem)
 
 if (GDAL_USE_POPPLER)
-  target_include_directories(gdal_PDF PRIVATE ${POPPLER_INCLUDE_DIRS} ${POPPLER_INCLUDE_DIRS}/../)
-  gdal_target_link_libraries(TARGET gdal_PDF LIBRARIES POPPLER::Poppler)
-  string(REGEX REPLACE "([0-9]+)\\.([0-9]+).*" "\\1" POPPLER_VERSION_MAJOR ${POPPLER_VERSION_STRING})
-  string(REGEX REPLACE "([0-9]+)\\.([0-9]+).*" "\\2" POPPLER_VERSION_MINOR ${POPPLER_VERSION_STRING})
+  target_include_directories(gdal_PDF PRIVATE ${Poppler_INCLUDE_DIRS} ${Poppler_INCLUDE_DIRS}/../)
```

Comment:
This line doesn't look healthy, given that `${Poppler_INCLUDE_DIRS}` might be a list of multiple paths. The var must contain all required paths. If it doesn't either the Find modules is broken, or the consuming source code. 
(If it is needed for transition, it formally still takes a loop to do this correctly.)

Reply:
yeah I agree this is not super clean currently. The issue here is that we must also cope with the source code that was meant for the autoconf & nmake build systems that assume that the include will be ${prefix}/include (the #includes are "poppler/foo.h"), while the CMake module returns ${prefix}/include/poppler in Poppler_INCLUDE_DIR
When we don't need to support the historical biuld systems, we'll be able to modify the source code and just use ${Poppler_INCLUDE_DIRS} 
Anyway in practice FindPoppler.cmake only returns a single path (unless the user would specify several ones, but that doesn't make sense)

Reply:
tracked as https://github.com/OSGeo/gdal/issues/5170